chromedriver_autoinstaller/__init__.py

```diff
@@ -7,7 +7,7 @@
 from . import utils
 
 
-def install(cwd: bool = False, path: Optional[AnyStr] = None, no_ssl: bool = False):
+def install(cwd: bool = False, path: Optional[AnyStr] = None, no_ssl: bool = False, make_version_dir: bool = True):
```

Comment:
I think this is a cool idea for people who don't need to keep outdated driver versions!

Reply:
@max32002 I created a [fork I am publishing to PyPi here. ](https://github.com/shawnCaza/python-chromedriver-autoupdater/tree/master)

Are you interested in submitting a PR on that fork for the 'make_version_dir' idea? Or would you mind if I added it in myself?